test/e2e/storage/vsphere/vsphere_volume_perf.go

```diff
@@ -86,7 +86,7 @@ var _ = utils.SIGDescribe("vcp-performance [Feature:vsphere]", func() {
 
 		nodes, err := e2enode.GetReadySchedulableNodes(ctx, client)
 		framework.ExpectNoError(err)
-		gomega.Expect(len(nodes.Items)).To(gomega.BeNumerically(">=", 1), "Requires at least %d nodes (not %d)", 2, len(nodes.Items))
```

Comment:
Note that the explanation does not match the code (off-by-one error). As the code is what actually runs, I kept that logic and updated the explanation accordingly.